sdk/servicebus/service-bus/review/service-bus.api.md

```diff
@@ -133,23 +189,23 @@ export interface PeekMessagesOptions extends OperationOptionsBase {
 
 // @public
 export interface QueueProperties {
-    authorizationRules?: AuthorizationRule[];
-    autoDeleteOnIdle?: string;
-    deadLetteringOnMessageExpiration?: boolean;
-    defaultMessageTtl?: string;
-    duplicateDetectionHistoryTimeWindow?: string;
-    enableBatchedOperations?: boolean;
-    enablePartitioning?: boolean;
-    forwardDeadLetteredMessagesTo?: string;
-    forwardTo?: string;
-    lockDuration?: string;
-    maxDeliveryCount?: number;
-    maxSizeInMegabytes?: number;
-    name: string;
-    requiresDuplicateDetection?: boolean;
-    requiresSession?: boolean;
-    status?: EntityStatus;
-    userMetadata?: string;
+    readonly authorizationRules?: AuthorizationRule[];
+    readonly autoDeleteOnIdle: string;
+    deadLetteringOnMessageExpiration: boolean;
+    defaultMessageTtl: string;
+    duplicateDetectionHistoryTimeWindow: string;
+    readonly enableBatchedOperations: boolean;
```

Comment:
Should these be readonly? I was able to update everything except for session/duplicateDetection/partitioning/name.

Reply:
Right now, I've assigned them according to the docs. 
https://docs.microsoft.com/en-us/rest/api/servicebus/update-queue

Should we talk to the service team?

Reply:
yes please

Reply:
From what Shankar(from the service team) said, this is a docs issue and we should allow all the properties that are updatable.

Reply:
Updated the updatable properties for update methods